src/checks.cc

```diff
@@ -349,10 +349,11 @@ LinterResult CheckJoin(absl::string_view sql, const LinterOptions &option) {
            LinterResult result;
            // SingleNodeDebugString also returns the type if there are any.
            // If it is equal to normal kind string, this means join is typeless.
+           int position = node->GetParseLocationRange().start().GetByteOffset();
```

Comment:
Nit: you're fetching a node location in few places. If it happens more than twice, you could write a tiny inlined helper functions `int GetStartPosition(const ASTNode&)`, `GetEndPosition` and the same for `const Token&`

Reply:
Ok.